src/ProductConstructionService/ProductConstructionService.ReproTool/ReproToolOptions.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using CommandLine;
+
+namespace ProductConstructionService.ReproTool;
+
+internal class ReproToolOptions
+{
+    [Option('s', "subscription", HelpText = "Subscription that's getting reproduced", Required = true)]
+    public required string Subscription { get; init; }
+
+    [Option("github-token", HelpText = "GitHub token", Required = true)]
+    public required string GitHubToken { get; init; }
+
+    [Option("commit", HelpText = "Commit to flow sha", Required = false)]
+    public string? Commit { get; init; }
```

Comment:
I think it should be possible to flow a specific build rather than just to trigger a subscription/for a commit.
Because we also want to flow the same assets, not just sources, so that we test updates to the version files too.

Reply:
You mean to do the normal, non source code? In that case maybe we could have a separate command for it, having something like `repro-tool subsciption --id .......` and `repro-tool build --id ... --type source/assets...`

Reply:
> You mean to do the normal, non source code?

No. To do the same one as we're reproing - sources and packages - this is the normal code flow. We only flow builds that contain assets and we update those assets in the version files. This is important specifically for the backflows.

I think we will definitely want to flow mostly builds - not subscriptions or latest SHAs of a repo - we will want to verify all of the changes that happen during a code flow. I think we should have 1 command only which can have `--subscription` and `--build` arguments.

Reply:
Ahh right, if we don't provide the build for backwards flow, we won't have a way of getting asset

Reply:
added support for flowing builds too